### PR TITLE
add binpacking best practice

### PR DESCRIPTION
**Pull request by melodyyangaws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/performance/docs/binpack.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+Quick Start:
+```bash
+eks-node-viewer --resources cpu,memory
+```
+
+## Install Bin-packing custom scheduler
+
+In the [scheduling-plugin](https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins) ` NodeResourcesFit` of kube-scheduler, there are two scoring strategies that support the bin packing of resources:       `MostAllocated` and `RequestedToCapacityRatio`. We created a custom scheduler based on the **MostAllocated strategy**. See the K8s’s [Resource Bin Packing](https://kubernetes.io/docs/concepts/scheduling-eviction/resource-bin-packing/) documentation for more details.
+
+The following customer scheduler named “my-scheduler” is created for EKS version v1.28, as the “kube-scheduler” container image version and “KubeSchedulerConfiguration“ API version are set for the v1.28. Please adjust them accordingly, if your EKS version is different.
```
Comment (lmouhib) [inline content/performance/docs/binpack.md]: I think we need to be more explicit that a user need to use an image/api version that match the k8s they are running.

File: content/performance/docs/binpack.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+## Validate the Custom Scheduler
+
+- **Step1:** Launch the node viewer in a terminal:
+```bash
+eks-node-viewer --resources cpu,memory
+```
+
+- **Step 2:** Add the custom scheduler to an EMR on EKS job via Pod Templates. For exmaple:
+
+**executor-pod-template.yaml (line #3)**
```
Comment (lmouhib) [inline content/performance/docs/binpack.md]: We can simplify and show that its also possible by just passing the scheduler in the spark conf, using `spark.kubernetes.scheduler.name`, without using the a pod template.

File: content/performance/docs/binpack.md (hunk 1 above)
Comment (melodyyangaws) [reply content/performance/docs/binpack.md]: see the [improve binpack](https://github.com/aws/aws-emr-containers-best-practices/pull/57/commits/55123ad158cd508fbc36ab7faedba6ef833598d4)

File: content/performance/docs/binpack.md (hunk 2 above)
Comment (melodyyangaws) [reply content/performance/docs/binpack.md]: added the extra example. see the 
[improve binpack](https://github.com/aws/aws-emr-containers-best-practices/pull/57/commits/55123ad158cd508fbc36ab7faedba6ef833598d4)

**Review by mattliemAWS — APPROVED** [review]
lgtm

**Pull request merged**